sjb/config/test_cases/test_pull_request_origin_web_console.yml

```diff
@@ -0,0 +1,6 @@
+---
+parent: 'test_cases/test_branch_origin_web_console.yml'
+overrides:
```

Comment:
To make this Rosie-compatible for `[test]` you will need to add:

```yml
overrides:
  test: 'origin-web-console'
```

As per [this doc](https://github.com/openshift/aos-cd-jobs/blob/master/sjb/syntax.md#merge-and-test).